### ISSUE TITLE
ferries not shown at zoom 9+

### ISSUE THREAD

**Issue by almccon** [stamen/toner-carto#11]
Looks like we want to grab route=ferry Some examples: http://www.openstreetmap.org/way/199895398 http://www.openstreetmap.org/way/166897068 http://www.openstreetmap.org/way/98771919

Since this would require modifying imposm3_mapping.json, I'm going to wait on this. It feels low priority to me, anyway.
